intake_erddap/utils.py

```diff
@@ -11,3 +18,67 @@ def get_project_version() -> str:
     from intake_erddap import __version__
 
     return __version__
+
+
+def return_category_options(
+    server: str,
+    category: Optional[str] = "standard_name",
+) -> DataFrame:
+    """Find category options for ERDDAP server.
+
+    Parameters
+    ----------
+    server : str
+        ERDDAP server address, for example: "http://erddap.sensors.ioos.us/erddap"
```

Comment:
Access to this erddap should be via `https`. If it still accepts `http` it shouldn't for very long.

Reply:
Should be fixed now.